src/grpc/grpc_server.cc

```diff
@@ -2366,6 +2366,12 @@ Server::Server(
     builder_.AddChannelArgument(
         GRPC_ARG_HTTP2_MAX_PING_STRIKES,
         keepalive_options.http2_max_ping_strikes_);
+    builder_.AddChannelArgument(
```

Comment:
can you just check if the parameter is populated and only `AddChannelArgument` if it is nonzero?

Reply:
Sorry I'm still new to this. Can you please provide more details on how to check whether the parameter is populated?

Reply:
Or do you mean I should make code changes such that AddChannelArgument will only be invoked when the parameter is set by the user with a non-default value?